webhook.go

```diff
@@ -365,9 +383,9 @@ func updateAnnotation(target map[string]string, added map[string]string) (patch
 
 // If return nil, no changes required
 func (whsvr *WebhookServer) mutateInitialization(pod corev1.Pod, req *v1beta1.AdmissionRequest) (*tsiMutateConfig, error) {
-	namespace := req.Namespace
-	if namespace == metav1.NamespaceNone {
-		namespace = metav1.NamespaceDefault
+	podNamespace := req.Namespace
+	if podNamespace == metav1.NamespaceNone {
+		podNamespace = metav1.NamespaceDefault
```

Comment:
if we have protected namespaces, etc. this should be done before the check for mutationRequired

Reply:
Looks like the podNamespace was actuallly never assigned. Fixed. Good catch! :)